### ISSUE TITLE
Is the path ID a variable-length integer or not?

### ISSUE THREAD

**Issue by nandsky** [quicwg/multipath#424]
As describe in [initial_max_path_id](https://github.com/quicwg/multipath/blob/7041135f5546cba33c971ea1ce0a747d953d3405/draft-ietf-quic-multipath.md?plain=1#L258)

> a variable-length integer specifying the maximum path identifier an endpoint is willing to maintain at connection initiation.

If the path ID is a variable-length integer, then the range of the path ID should be from 0 to 2^30-1, because the first two bits represent the integer length.  However, the current draft specifies a range of 0 to 2^32-1.

> This extension specifies a new path identifier (Path ID), which is an
integer between 0 and 2^32-1 (inclusive).

**Comment by mirjak** [comment]
Yes, the Path ID is limited to a maximum value of 2^32-1 because of its usage in the none calculation, see https://quicwg.org/multipath/draft-ietf-quic-multipath.html#name-nonce-calculation
